src/aggregate/aggregate_request.c

```diff
@@ -568,7 +568,13 @@ static int parseQueryArgs(ArgsCursor *ac, AREQ *req, RSSearchOptions *searchOpts
   searchOpts->ninkeys = inKeys.argc;
   searchOpts->legacy.infields = (const char **)inFields.objs;
   searchOpts->legacy.ninfields = inFields.argc;
-  searchOpts->language = RSLanguage_Find(languageStr, 0);
+  // if language is NULL, set it to UNDEFINED and it will be updated later,
+  // taking the index language
+  if(languageStr == NULL) {
+    searchOpts->language = RS_LANG_INDEX_DEFAULT;
```

Comment:
The doc say `UNDEFINED`, but the code says `DEFAULT`.

Reply:
OK, I updated the comment. Thanks